### PR TITLE
Add seed parameter.

### PR DESCRIPTION
**Pull request by gerritjandebruin**
Hi Elior Cohen (and others),

This is my first pull request ever. Please help me a bit when I miss certain things.

A while ago we noticed that this Node2Vec implementation yields different results, even when run in one Python process. This problem is also mentioned by @marcovarrone in Issue #26.

In this pull request, we (@pereirabarataap and me) propose to add a simple parameter `seed`, which allows to set the random seed of both Numpy and the Python random module.

This issue fixes:
- [x] When calling Node2Vec with the same parameters, will obtain the exact same embedding in one process. We show this in the `another-example.py` at https://github.com/gerritjandebruin/node2vec/blob/another-example/another-example.py.

Future work:
- [ ] Make the seed also work with multiple workers.
- [ ] Make the seed work between different Python process. 
      At this moment it does not, also not with a fixed PYTHONHASHSEED.

Kind regards,

Gerrit-Jan and António

### CODE REVIEW

**Comment by eliorc** [comment]
If I understand you correctly - do you mean this works only for `workers=1`?

**Comment by gerritjandebruin** [comment]
Yes. This is also the case for gensim (https://radimrehurek.com/gensim/models/word2vec.html).

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ edges_kv.save_word2vec_format(EDGES_EMBEDDING_FILENAME)
         Use these keys exactly. If not set, will use the global ones which were passed on the object initialization`
     10. `quiet`: Boolean controlling the verbosity. (default: False)
     11. `temp_folder`: String path pointing to folder to save a shared memory copy of the graph - Supply when working on graphs that are too big to fit in memory during algorithm execution.
+    12. `seed`: Seed for the random number generator.
```
Comment (eliorc) [inline README.md:76]: Please add an example for using this parameter in the README, also please include some information in the README about this only working on single worker

**Comment by eliorc** [comment]
Hi, please change the target of the request to the `dev` branch, I want to include some changes of my own @gerritjandebruin

File: README.md (hunk 1 above)
Comment (gerritjandebruin) [reply README.md]: This is fixed in the latest commit.

**Review by gerritjandebruin — COMMENTED** [review]
These two commits fix a uncorrelated typo and add additional explanation about the seed parameter.

The working of the seed parameter can be checked with the following example:
```
import random

import networkx as nx
from node2vec import Node2Vec
import numpy as np
import pandas as pd

random.seed(0)
np.random.seed(0)

G = nx.karate_club_graph()

def get_embedding():
    model = Node2Vec(G, dimensions=4, seed=0)
    embedding = model.fit()
    return pd.DataFrame(embedding.wv.vectors, index=embedding.wv.index2entity)

print('Embedding 1')
print(get_embedding().sort_index().head())

print('Embedding 2')
print(get_embedding().sort_index().head())
```

**Review by eliorc — APPROVED** [review]

**Pull request merged**